packages/mui-material/src/styles/components.d.ts

```diff
@@ -380,6 +380,8 @@ export interface Components<Theme = unknown> {
   };
   MuiPopper?: {
     defaultProps?: ComponentsProps['MuiPopper'];
+    styleOverrides?: ComponentsOverrides<Theme>['MuiPopper'];
+    variants?: ComponentsVariants['MuiPopper'];
```

Comment:
`variants` doesn't make sense for Popper component.
```suggestion
```

Reply:
Hello, thank you for your input. Will do.

One question, what are your thought on the test failing in argos? Can't quite figure out why the change occurs on the graph.

Reply:
It's flaky and not related to this PR. You can ignore it.